contracts/contracts/link_token.cairo

```diff
@@ -0,0 +1,83 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.math import assert_not_zero, assert_le
+from starkware.cairo.common.uint256 import Uint256
+from starkware.starknet.common.syscalls import get_caller_address
+from starkware.cairo.common.bool import FALSE, TRUE
+from openzeppelin.token.erc20.library import ERC20
+from openzeppelin.token.erc20.presets.ERC20Upgradeable import (
+    initializer,
+    upgrade,
+    name,
+    symbol,
+    totalSupply,
+    decimals,
+    balanceOf,
+    allowance,
+    transfer,
```

Comment:
Currently this contract can transfer token to himself. I  I can delete  `transfer` and `approve` from `from openzeppelin.token.erc20.presets.ERC20Upgradeable import`   and rewrite those two function to check if `recipient != get_contract_address()`

Reply:
@krebernisak do we want this behaviour? In the Solidity version: https://github.com/smartcontractkit/LinkToken/blob/f307ea6d4c02dd87ea4c1f1032e3e384ceac20c2/contracts/v0.6/LinkToken.sol#L38